modMcf/src/org/aion/mcf/db/ContractDetailsCacheImpl.java

```diff
@@ -103,7 +103,7 @@ public ByteArrayWrapper get(ByteArrayWrapper key) {
      */
     @Override
     public byte[] getStorageHash() {
-        return origContract.getStorageHash();
+        return origContract == null ? null : origContract.getStorageHash();
```

Comment:
already applied in #842

Reply:
Merged #842 to `mpm` and rebased from `mpm`